### PR TITLE
Adding Jackett

### PR DESCRIPTION
**Pull request by jesse-troy**
I have a lot of features I want to add that I have running on my own system, but I am not at all familiar with using git (or github) so I am submitting this very simple feature first.

Jackett assists media sourcing applications (sonarr, radarr, etc) with searching torrent trackers. 

Please let me know if I should be doing this any differently. I will also message you directly on gitter.

### CODE REVIEW

File: nas.yml [hunk 1]
```diff
@@ -130,3 +130,7 @@
   - import_tasks: tasks/get_iplayer.yml
     when: (get_iplayer_enabled | default(False))
     tags: get_iplayer
+
+  - import_tasks: tasks/jackett.yml
+    when: jackett_enabled
```
Comment (davestephens) [inline nas.yml]: Please can you make the `when:` clause include a default as with the other blocks

**Comment by davestephens** [comment]
This is awesome, thank you!

Only thing missing is adding it to the README, and a minor tweak required on the nas.yml. 

Aside from that, good to go :+1:

File: nas.yml (hunk 1 above)
Comment (jesse-troy) [reply nas.yml]: @davestephens I have made this change, but I don't know how to update this pull request. Could you point me in the right direction?

Comment (jesse-troy) [reply nas.yml]: I have also updated the README with the following line, but for some reason it is not showing up here. 
 `* [Jackett] (https://github.com/Jackett/Jackett) - API Support for your favorite torrent trackers`

Git/Github is very confusing to me for some reason.

Comment (davestephens) [reply nas.yml]: Looks good to me - you can see all of the changes in the diff view: https://github.com/davestephens/ansible-nas/pull/67/files

**Comment by davestephens** [comment]
Merged! 😎

**Pull request merged**